packages/ui/src/documents/CreateDocument.tsx

```diff
@@ -119,7 +120,7 @@ export const CreateDocument: FunctionComponent<Props> = (props) => {
       renderHeader={() => {
         return <SecondaryHeader>
           <Box direction="row" gap="small" align="center">
-            <Link to={documentRoutes.index} size="large">
+            <Link to={documentRoutes.index}>
```

Comment:
Bug, This was generating typescript errors for because Link that not have that prop